book/src/libs/havoc.md

```diff
@@ -1,211 +1,41 @@
+[![Go Reference](https://pkg.go.dev/badge/github.com/smartcontractkit/chainlink-testing-framework/havoc.svg)](https://pkg.go.dev/github.com/smartcontractkit/chainlink-testing-framework/havoc)
 
-### Features
+The `havoc` package is designed to facilitate chaos testing within [Kubernetes](https://kubernetes.io/) environments using [Chaos Mesh](https://chaos-mesh.org/). It offers a structured way to define, execute, and manage chaos experiments as code, directly integrated into Go applications or testing suites, simplifying the creation and control of Chaos Mesh experiments.
 
-- **Chaos Object Management:** Easily create, update, pause, resume, and delete chaos experiments using Go structures and methods.
-- **Lifecycle Hooks:** Utilize chaos listeners to hook into lifecycle events of chaos experiments, such as creation, start, pause, resume, and finish.
-- **Support for Various Chaos Experiments:** Create and manage different types of chaos experiments like NetworkChaos, IOChaos, StressChaos, PodChaos, and HTTPChaos.
-- **Chaos Experiment Status Monitoring:** Monitor and react to the status of chaos experiments programmatically.
+## Features
 
-### Installation
+- **Chaos Object Management:** Create, update, pause, resume, and delete chaos experiments using Go structures and methods.
+- **Lifecycle Hooks:** Utilize chaos listeners to hook into the lifecycle of chaos experiments.
+- **Different Experiments:** Create and manage different types of chaos experiments to affect network, IO, K8s pods, and more.
+- **Active Monitoring:** Monitor and react to the status of chaos experiments programmatically.
 
-To use `havoc` in your project, ensure you have a Go environment setup. Then, install the package using go get:
+## Requirements
 
-```
-go get -u github.com/smartcontractkit/chainlink-testing-framework/havoc
-```
-
-Ensure your Kubernetes cluster is accessible and that you have Chaos Mesh installed and configured.
-
-### Monitoring and Observability in Chaos Experiments
-
-`havoc` enhances chaos experiment observability through structured logging and Grafana annotations, facilitated by implementing the ChaosListener interface. This approach allows for detailed monitoring, debugging, and visual representation of chaos experiments' impact.
-
-#### Structured Logging with ChaosLogger
+- [Go](https://go.dev/)
+- A Kubernetes cluster with [Chaos Mesh installed](https://chaos-mesh.org/docs/quick-start/)
 
-`ChaosLogger` leverages the zerolog library to provide structured, queryable logging of chaos events. It automatically logs key lifecycle events such as creation, start, pause, and termination of chaos experiments, including detailed contextual information.
+## Installation
 
-Instantiate `ChaosLogger` and register it as a listener to your chaos experiments:
-
-```
-logger := havoc.NewChaosLogger()
-chaos.AddListener(logger)
+```sh
+go get -u github.com/smartcontractkit/chainlink-testing-framework/havoc
```

Comment:
Maybe just `go get`? Updating packages is on user, maybe I don't need it

Reply:
I'd argue it's unnecessary most of the time. if you're familiar with Go, this isn't exactly new.